internal/rest/rest_test.go

```diff
@@ -131,9 +155,8 @@ func TestClient_NewRequest(t *testing.T) {
 			t.Parallel()
 
 			cli := rest.NewClient(nil, c.Root)
-			req, err := cli.NewRequest(c.Method, c.Path, c.Body, c.Query)
-			assert.Equals(t, err, c.Err)
-			assert.Equals(t, req, c.Request)
+			req, err := cli.NewRequest(context.TODO(), c.Method, c.Path, c.Body, c.Query)
```

Comment:
I think context.Background() as per https://golang.org/pkg/context/#Background is idiomatic

Applies to most of the tests in this file.

Reply:
Nice spot, Tim. I've updated the tests to only use `context.Background` -- https://github.com/senseyeio/mbgo/pull/11/commits/4c92a0839210e895bc0c06b31ed9e66e7afc6cb7.